frontend/src/utils/serializeQueryParams.ts

```diff
@@ -49,13 +49,18 @@ const decodeFilterItem = (
         return undefined;
     }
 
-    const [operator, values = ''] = input.split(':');
-    if (!operator) return undefined;
+    const pattern =
```

Comment:
Make serialization have same logic as backend. Only splitting on operators, not if text has colon in it.